dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/ClientWindowScopedTest.war/src/io/openliberty/org/apache/faces40/fat/client/window/FlowBean.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.org.apache.faces40.fat.client.window;
+
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.faces.context.FacesContext;
+import jakarta.faces.lifecycle.ClientWindow;
+import jakarta.inject.Named;
+
+@Named(value = "flowBean")
+@RequestScoped
```

Comment:
Shouldn't this test use `@ClientWindowScoped` on the beans?

Reply:
By switching it, I get : `jakarta.el.PropertyNotFoundException: Target Unreachable, identifier [flowBean] resolved to null` 

I need to spend some time learning about `ClientWindowScoped` as I don't fully understand this change.

Reply:
Or we might have a bug.

Reply:
I think the exception above was because the beans weren't serializable.  @ClientWindowScoped works fine for me now.